replica_aggregator/src/main/scala/uk/ac/wellcome/platform/storage/replica_aggregator/models/ReplicationAggregationSummary.scala

```diff
@@ -5,7 +5,7 @@ import java.time.Instant
 import uk.ac.wellcome.platform.archive.common.operation.models.Summary
 
 sealed trait ReplicationAggregationSummary extends Summary {
-  val replicationSet: ReplicationSet
+  val replicaPath: String
```

Comment:
Value class would be slightly better than `String` (preventing certain modes of future misuse), and should incur no performance penalty. As per: https://docs.scala-lang.org/overviews/core/value-classes.html

```
class ReplicaPath(val underlying: String) extends AnyVal
```